### PR TITLE
[SPARK-39205][PYTHON][PS][INFRA] Add `PANDAS API ON SPARK` label

### PR DESCRIPTION
**Pull request by Yikun**
### What changes were proposed in this pull request?
Add `PANDAS API ON SPARK` label

### Why are the changes needed?
Add `PANDAS API ON SPARK` label

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
CI passed

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
Oh! you were faster than me!

File: .github/labeler.yml [hunk 1]
```diff
@@ -130,6 +130,8 @@ STRUCTURED STREAMING:
 PYTHON:
   - "bin/pyspark*"
   - "**/python/**/*"
+PANDAS API ON SPARK:
```
Comment (HyukjinKwon) [inline .github/labeler.yml:133]: Oh yeah. I think this label name is correct and more consistent than PS

**Review by HyukjinKwon — APPROVED** [review]

**Review by yaooqinn — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Merged to master.

**Review by zhengruifeng — APPROVED** [review]

**Comment by Yikun** [comment]
> Oh! you were faster than me!

lol, just for explain clearly in mail list

**Pull request closed without merging**